content/en/blog/2021-05-31-june-update.md

```diff
@@ -0,0 +1,208 @@
+involved. Read last month's update [here](/blog/2021/04/may-2021-update).
+
+Let's recap what happened in May - there has been so much happening!
+
+## Newest Flux release
+
+With the 0.14 release of Flux we added a number of big features to Flux.
+
+The highlights are:
+
+- We replaced the Flux monitoring stack with Prometheus Operator and
+  kube-prometheus-stack.
+- Support for a wider range of SSH key formats (CLI)
+- Send Flux events to Azure Event Hub (notification-controller)
+- Support for a wider range of OpenPGP key formats (source-controller)
+- Reduce the memory usage when cloning repositories with large files
+  (source-controller)
+- Support for including the content of app repos into the main config
+  one (better alternative to git submodules and kustomize remote
+  bases) (source-controller)
+
+Docs:
+
+- 📔 [GitRepository include
+  documentation](/docs/components/source/gitrepositories/#including-gitrepository)
+- 🎁 Checkout the new [guide on ways of structuring your
+  repositories](/docs/guides/repository-structure/)
+
+If you are entirely new to Flux, check out our [get started
+guide](/docs/get-started/). If you are on
+an older version of Flux, check out our [migration
+documentation](/docs/migration/).
+
+## Flagger 1.11 is here
+
+Over in the Flagger project, since our last blog the versions 1.9, 1.10 and
+1.11 were released and they bring a number of nice new features and
+improvements.
+
+1.9.0 came with improvements to the Gloo Edge integration:
+
+- Starting with this version, Flagger no longer requires Gloo
+  discovery to be enabled. Flagger generated the Gloo upstream
+  objects on its own and optionally it can use an existing upstream
+  (specified with .spec.upstreamRef) as a template.
+- Also: Adjusted Nginx ingress canary headers on init and promotion
+
+1.10.0 comes with support for Graphite metric templates, additionally:
+
+- ConfigTracker: Scan envFrom in init-containers
+- e2e: Update Istio to v1.10 and Contour to v1.15
+
+:warning: Flagger 1.11.0 comes with a breaking change: the minimum supported
+version of Kubernetes is v1.19.0.
+
+This release comes with support for Kubernetes Ingress `networking.k8s.io/v1`.
+The Ingress from `networking.k8s.io/v1beta1` is no longer supported, affected
+integrations: NGINX and Skipper ingress controllers.
+
+## KubeCon round-up
+
+KubeCon 2021 Europe happened in early May and the Flux team was happy to
+represent. If you entirely missed the event, here are a couple of links
+we'd like to share:
+
+- 05 May 2021 - [Keynote: CNCF Project Update: Flux - Stefan
+  Prodan](https://youtu.be/kuzZpKHLoVM) at KubeCon 2021
+  Europe
+- 06 May 2021 - [Flux: Multi-tenancy Deep Dive - Philip
+  Laine](https://youtu.be/F7B_TBcIyl8) at KubeCon 2021
+  Europe
+- 04 - 07 May 2021 - [Flux Pavilion at KubeCon EU! Talks and demos
+  every 30 minutes! Say
+  hello!](https://bit.ly/Flux_KubeConEU2021)
+
+These are recordings of talks which were given by Flux maintainers.
```

Comment:
Do we want to add all Kubecon talks by Flux maintainers? If we just want to list a few of them, that is fine, we can probably remove this line entirely

Reply:
Do you have more links?

Reply:
I went with what's on https://fluxcd.io/community/#talks

Reply:
I know we listed them all in the May blog 🙂